docs/framework/wpf/app-development/how-to-determine-the-installed-version-of-wpf.md

```diff
@@ -1,21 +0,0 @@
----
-title: "How to: Determine the Installed Version of WPF"
-ms.date: "03/30/2017"
-helpviewer_keywords: 
-  - "version [WPF], finding"
-ms.assetid: 99971cef-e218-4f9f-a4c1-350332741860
----
-# How to: Determine the Installed Version of WPF
```

Comment:
Should the links to this article in TOC and the index page be removed as well? Or they are kept for redirects?

Reply:
I forgot about those. However, I don't even know if they should be removed or kept for redirects. so, I'll wait until this is reviewed.

Reply:
They should be removed.

Reply:
I found these references:

```
wpf\app-development\build-and-deploy-how-to-topics.md:17: [Determine the Installed Version of WPF](how-to-determine-the-installed-version-of-wpf.md)
wpf\app-development\toc.yml:134:        href: how-to-determine-the-installed-version-of-wpf.md
```

Just update those links to the new one and you're good to go!